es6.md

```diff
@@ -89,3 +89,52 @@ interface ArrayPattern <: Pattern {
     elements: [ Pattern | null ];
 }
 ```
+
+# Classes
+
+```js
+interface Class <: Node {
+    id: Identifier | null;
+    superClass: Expression;
+    body: ClassBody;
+}
+```
+
+## ClassBody
+
+```js
+interface ClassBody <: Node {
+    type: "ClassBody";
+    body: [ MethodDefinition ];
+}
+```
+
+## MethodDefinition
+
+```js
+interface MethodDefinition <: Node {
+    type: "MethodDefinition";
+    key: Identifier;
+    value: FunctionExpression;
+    kind: "init" | "get" | "set";
```

Comment:
@sebmck Isn't it `""` for regular methods in current implementation? `init` seems to make sense for `Property` only (where you actually initialize it with some value), however I'm more than open to specifying `"method"` or smth here.